recipe/meta.yaml

```diff
@@ -29,7 +29,7 @@ requirements:
     - python
     - pywin32-ctypes  # [win]
     - secretstorage >=3.2  # [linux]
-    - importlib_metadata >=3.6
+    - importlib_metadata  # [py<310]
```

Comment:
@ccordoba12, I don't quite follow why the `>=3.6` constraint was dropped here.  It seems to still be needed upstream:
https://github.com/jaraco/keyring/blob/v23.6.0/setup.cfg#L26

I think this is causing `pip check` to fail for us:

Reply:
I think we might need to have this build marked as broken or patched, since we might pick up this version in `poetry` even if build 1 fixes this constriant.

Reply:
> I don't quite follow why the >=3.6 constraint was dropped here. It seems to still be needed upstream

Sorry, perhaps I thought that `3.6` was related to the Python version and that's why I removed it (I can't recall right now).